src/components/CaptureMatching/CaptureMatchingView.js

```diff
@@ -234,13 +235,73 @@ const useStyle = makeStyles((theme) => ({
   growerBox2: {},
 }));
 
+const filterTree = (trees, capture) => {
+  log.warn('capture:', capture);
+  log.warn('trees:', trees);
+  if (!capture) return [];
+  const capturedAt = new Date(capture.captured_at);
+  const filteredTrees = trees.filter((tree) => {
```

Comment:
@dadiorchen Can we remove this filter if it's now being performed on the API side?